microsoft-edge/extensions-chromium/publish/auto-update.md

```diff
@@ -22,18 +22,16 @@ ms.date: 04/13/2021
    limitations under the License.  -->
 # Automatically update extensions in Microsoft Edge
 
-When you set your extension to automatically update, your extension shares the following benefits with Microsoft Edge when set to automatically update.
+> [!NOTE]
+> This article doesn't apply to extensions that you publish using the [Partner Center](https://partner.microsoft.com/dashboard/microsoftedge/public/login?ref=dd) dashboard.  You can use the dashboard to release updated versions to your users and to the Microsoft Edge Add-ons website.  For more information, see [Update a Microsoft Edge extension](../publish/update-extension.md).
```

Comment:
we can merge this document for now, but separately we should review this document with engineering team for its relevance.